packages/taquito-michel-codec/michelson.json

```diff
@@ -2853,187 +2728,258 @@
     },
     "bls12_381_fr": {
       "ty": "bls12_381_fr",
       "ty_args": "bls12_381_fr",
-      "related_ops": [ "ADD", "INT", "MUL", "NEG" ]
+      "related_ops": ["ADD", "INT", "MUL", "NEG"]
     },
     "string": {
       "ty": "string",
       "ty_args": "string",
-      "related_ops": [ "CONCAT", "SIZE", "SLICE" ]
+      "related_ops": ["CONCAT", "SIZE", "SLICE"]
     },
     "nat": {
       "ty": "nat",
       "ty_args": "nat",
       "related_ops": [
-        "ABS", "ADD", "AND", "EDIV", "INT", "ISNAT", "LEVEL", "LSL", "LSR",
-        "MUL", "NEG", "NOT", "OR", "READ_TICKET", "SIZE", "SLICE",
-        "SPLIT_TICKET", "SUB", "TICKET", "TOTAL_VOTING_POWER",
-        "VOTING_POWER", "XOR"
+        "ABS",
+        "ADD",
+        "AND",
+        "EDIV",
+        "INT",
+        "ISNAT",
+        "LEVEL",
+        "LSL",
+        "LSR",
+        "MUL",
+        "NEG",
+        "NOT",
+        "OR",
+        "READ_TICKET",
+        "SIZE",
+        "SLICE",
+        "SPLIT_TICKET",
+        "SUB",
+        "TICKET",
+        "TOTAL_VOTING_POWER",
+        "VOTING_POWER",
+        "XOR"
       ]
     },
     "int": {
       "ty": "int",
       "ty_args": "int",
       "related_ops": [
-        "ABS", "ADD", "AND", "COMPARE", "EDIV", "EQ", "GE", "GT", "INT",
-        "ISNAT", "LE", "LT", "MUL", "NEG", "NEQ", "NOT",
-        "SAPLING_VERIFY_UPDATE", "SUB"
+        "ABS",
+        "ADD",
+        "AND",
+        "COMPARE",
+        "EDIV",
+        "EQ",
+        "GE",
+        "GT",
+        "INT",
+        "ISNAT",
+        "LE",
+        "LT",
+        "MUL",
+        "NEG",
+        "NEQ",
+        "NOT",
+        "SAPLING_VERIFY_UPDATE",
+        "SUB"
       ]
     },
     "bytes": {
       "ty": "bytes",
       "ty_args": "bytes",
       "related_ops": [
-        "BLAKE2B", "CHECK_SIGNATURE", "CONCAT", "KECCAK", "PACK", "SHA256",
-        "SHA3", "SHA512", "SIZE", "SLICE", "UNPACK"
+        "BLAKE2B",
+        "CHECK_SIGNATURE",
+        "CONCAT",
+        "KECCAK",
+        "PACK",
+        "SHA256",
+        "SHA3",
+        "SHA512",
+        "SIZE",
+        "SLICE",
+        "UNPACK"
       ]
     },
     "bool": {
       "ty": "bool",
       "ty_args": "bool",
       "related_ops": [
-        "AND", "CHECK_SIGNATURE", "EQ", "GE", "GT", "IF", "LE", "LOOP", "LT",
-        "MEM", "NEQ", "NOT", "OR", "PAIRING_CHECK", "UPDATE", "XOR"
+        "AND",
+        "CHECK_SIGNATURE",
+        "EQ",
+        "GE",
+        "GT",
+        "IF",
+        "LE",
+        "LOOP",
+        "LT",
+        "MEM",
+        "NEQ",
+        "NOT",
+        "OR",
+        "PAIRING_CHECK",
+        "UPDATE",
+        "XOR"
       ]
     },
     "mutez": {
       "ty": "mutez",
       "ty_args": "mutez",
       "related_ops": [
-        "ADD", "AMOUNT", "BALANCE", "CREATE_CONTRACT", "EDIV", "MUL", "SUB",
+        "ADD",
+        "AMOUNT",
+        "BALANCE",
+        "CREATE_CONTRACT",
+        "EDIV",
+        "MUL",
+        "SUB",
```

Comment:
The SUB instruction on type mutez is deprecated

Reply:
Even if it is deprecated, does that mean it should be forbidden here? I'm not sure it's our place to prevent people from using it?

Reply:
I could not find where this file is used in the code base, so I thought it was there for informative purposes only (Correct me if I am wrong please). I  pointed out the deprecated instruction so we keep the file up to date.

Reply:
It's deprecated, see
```
 const s = ProtoGreaterOfEqual(proto, Protocol.PsiThaCa)
          ? args(0, ['nat', 'int', 'timestamp'], ['nat', 'int', 'timestamp'])
          : args(0, ['nat', 'int', 'timestamp', 'mutez'], ['nat', 'int', 'timestamp', 'mutez']);
```

And `michelson.json` was left there at some point for reference but it's never used